CHANGELOG.md

```diff
@@ -1,7 +1,19 @@
 # Change Log
 
-## [v2.32.0](https://github.com/sensu/sensu-puppet/tree/v2.32.0)
+## [v2.33.0](https://github.com/sensu/sensu-puppet/tree/v2.33.0)
 
+[Full Changelog](https://github.com/sensu/sensu-puppet/compare/v2.32.0...v2.33.0)
```

Comment:
This shouldnt show up in the commit. It's because your master branch was not up to date. Could you please remove it.